public/app/features/alerting/unified/state/AlertingQueryRunner.ts

```diff
@@ -41,12 +42,24 @@ export class AlertingQueryRunner {
     return this.subject.asObservable();
   }
 
-  run(queries: AlertQuery[]) {
+  async run(queries: AlertQuery[]) {
```

Comment:
I think we should check every invocation of `AlertingQueryRunner.run()` to make sure that when `dataSourceSrv.get` throws we are not crashing the interface or make sure we have a `try..catch` mechanism here.